### DISCUSSION TITLE
Graphic error in Visual Studio Code

### DISCUSSION THREAD

**Discussion by RAPCCLOTHING** [community/community#132490] [Programming Help]
Please, i need ur help...
My VSC program started having these really annoying graphic issues a few months ago.
I tried fixing it by uninstalling and reinstalling the program, but it's still the same.
Has anyone else experienced this? If so, how did you fix it?
**(Note: the effect disappears when I hover the cursor over the affected areas, but it comes back after a few seconds.)**

![image](https://github.com/user-attachments/assets/66601f8c-2833-4833-b9a7-ad9ee67efd40)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by NewBumpEr** [comment]
Hello,

If you have an Nvidia GPU, try turning off Low Latency Mode. If the problem persists, disable hardware acceleration in Visual Studio Code. Also, if you're using a virtual machine, go to your display settings and uncheck "Enable 3D Acceleration." Finally, you can also try running Visual Studio code with the --disable-gpu flag.

Hope this helps with your graphics issues. I will be glad if my answer helps to solve your problem!

**Comment by aishamc** [comment]
Hey there!
One thing you might try is updating your graphics drivers, as outdated drivers can sometimes cause issues. Also, try checking for updates for Visual Studio Code itself, as newer versions might have fixed this bug.

If that doesn't work, go into the settings of Visual Studio Code and turn off any experimental features you might have enabled. You can also try changing the rendering engine by setting "disable-hardware-acceleration": true in the settings.

Another thing to consider is running VSC in compatibility mode for an older version of Windows if you're on a PC, or tweaking your system's display settings to see if that helps.

Hope one of these tips sorts it out for you!

**Comment by erenyavuzs** [comment]
Hi,

did you go into the some settings and change it or did it just happen on its own?

did you ever change the windows dpi settings?

try updating your graphics card
